core/unit_test/tstNeighborList.hpp

```diff
@@ -662,7 +662,7 @@ void testVerletListFullPartialRange()
         Cabana::slice<0>( aosoa ), 0, num_ignore, test_radius, cell_size_ratio,
         grid_min, grid_max );
 #else
-    (void)cell_size_ratio;
+    std::ignore = cell_size_ratio;
```

Comment:
Maybe just move the definition of `cell_size_ratio` into the `#if` block above because it is a parameter specific to the `VerletList`?

Reply:
Warnings are gone as far as I can tell